src/store/cart.js

```diff
@@ -1,25 +1,59 @@
 import axios from 'axios';
 
+const SET_CART = 'SET_CART';
+const UPDATE_LINEITEM_Q = 'UPDATE_CART';
+const DELETE_LINEITEM = 'DELETE_LINEITEM';
+
```

Comment:
I think we should separate the ACTIONS (above), from its handlers (below) and put them in different files) let me know what you think